### ISSUE TITLE
code does not match doc for extending optimizer with weight decay

### ISSUE THREAD

**Issue by ben-arnao** [tensorflow/addons#1700]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): n/a
- TensorFlow version and how it was installed (source or binary): current in github
- TensorFlow-Addons version and how it was installed (source or binary): n/a
- Python version: n/a
- Is GPU used? (yes/no): n/a

**Describe the bug**

The docs for decoupling weight decay extension do not match the library.

**Code to reproduce the issue**

n/a

**Other info / logs**

In the docs (https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/weight_decay_optimizers/DecoupledWeightDecayExtension) we see that an example is given to extend an existing optimizer.

"extend_with_decoupled_weight_decay( OptimizerX, weight_decay=weight_decay)"

How in the code the signature for this method is "def extend_with_decoupled_weight_decay(base_optimizer):"

There is no keyword argument for weight_decay

**Comment by WindQAQ** [comment]
I believe doc is wrong so you are more than welcome to issue a pull request to fix it :-) Thanks for pointing out!

**Comment by yil532** [comment]
Hello:) Do we still need help on this? @WindQAQ

**Comment by seanpmorgan** [comment]
Hi @yil532 it was fixed with #1718 .. thanks for bringing this up though we can close this issue.
